include/onnxruntime/core/session/onnxruntime_common_enums.h

```diff
@@ -0,0 +1,18 @@
+// Shared public enums used by both onnxruntime_c_api.h and onnxruntime_ep_c_api.h
```

Comment:
I believe this should not be necessary because onnxruntime_c_api.h includes onnxruntime_ep_c_api.h inline (end of file). The EP author is meant to include onnxruntime_c_api.h.

OrtStatus is another example of a symbol that is defined in onnxruntime_c_api.h but used in onnxruntime_ep_c_api.h

Reply:
I unfortunately couldn't get the branch building until I made this change- the new API is in onnxruntime_c_api.h (maybe that is not the right place for it? In this scenario, the caller of the API is the developer of an application); and since I want the API to return a compatibility status enum value, the symbol needs to already be available.